db/migrate/20170719210832_move_versions_table.rb

```diff
@@ -0,0 +1,5 @@
+class MoveVersionsTable < ActiveRecord::Migration
+  def change
+    rename_table :versions, :vestal_versions
```

Comment:
We should probably also test this against the version of mariadb DC is on and mysql UIC is on in terms of automatically renaming indexes and foreign keys. That bit us recently.